gister/gister.py

```diff
@@ -169,14 +173,24 @@ def create_gist():
     else:
         url = public_gist_url(conf, args)
         token_name = 'public_oauth'
+    description = 'created by github.com/tr3buchet/gister'
+    if args.description:
+        description = args.description
 
     headers = get_headers(conf, token_name) if not args.anonymous else None
-    payload = {'description': 'created by github.com/tr3buchet/gister',
+    payload = {'description': description,
                'public': not args.secret,
                'files': payload}
 #               'files': dict((k, {'content': v}) for k, v in payload)}
 #               'files': {payload[0]: {'content': payload[1]}}}
-    r = requests.post(url + '/gists', data=json.dumps(payload),
+    try:
+        r = requests.post(url + '/gists', data=json.dumps(payload),
                       headers=headers)
+    except Exception as e:
+        print(e)
+        sys.exit()
     r.raise_for_status()
-    print r.json()['html_url']
+    print(r.json()['html_url'])
+
+if __name__ == '__main__':
```

Comment:
the `__main__` block isn't needed because `create_gist()` is a console script entry point

also pep8 spacing issue, you should check out flake8 or even configure vim to use it